lib/sycamore/sycamore/transforms/llm_query.py

```diff
@@ -44,6 +45,7 @@ def __init__(
         number_of_elements: Optional[int] = None,
         llm_kwargs: dict = {},
         per_element: bool = True,
+        element_type: str = "",
```

Comment:
I don't think an empty string is the right default value if you want to allow anything. Probably you want `Optional[str] = None`.